include/vcpkg/base/messages.h

```diff
@@ -510,6 +510,24 @@ namespace vcpkg
                     "An example of env_var is \"HTTP(S)_PROXY\""
                     "'--' at the beginning must be preserved",
                     "-- Automatically setting {env_var} environment variables to \"{url}\".");
+    DECLARE_MESSAGE(
+        BinaryParagraphDescription,
+        (msg::spec,
+         msg::version,
+         msg::version,
+         msg::value,
```

Comment:
Ditto from somewhere else I don't think you can reuse replacements like this. Between this and this PR being draft not going to review further for now...